.circleci/config.yml

```diff
@@ -146,3 +157,48 @@ jobs:
                     --attach build/distributions/git-machete-intellij-plugin-*.zip \
                     --message "$tag"$'\n\n'"$change_notes" \
                     $tag
+
+
+  updateEapBuildNumber:
+    executor: docker_executor
+    steps:
+      - checkout
+      - reconfigure_origin_remote
+      - run:
+          name: Configure github user information
+          command: |
+            git config user.email "gitmachete@virtuslab.com"
+            git config user.name "Git Machete Bot"
+      - run:
+          # language=sh
+          command: |
+            if ./gradlew updateEapBuildNumber --exit-code ; then
```

Comment:
Nit: you can rework
```shell
if foo; then
  ....
  if bar; then
    ...
  fi
fi
```

to 

``shell
  if ! foo; then exit 0; fi
  ....
  if ! bar; then exit 0; fi
  ....
```

That's gonna be "flatter" and presumably easier to read... up to you tho